### PR TITLE
joe - 3rd party product changes

### PR DESCRIPTION
**Pull request by jartuso**
Added a new page for new ceegrids product, plus cleaned up out of date hegduino and wifi shield info.

### CODE REVIEW

File: website/docs/ThirdParty/ThirdPartyLanding.md [hunk 1]
```diff
@@ -14,14 +14,6 @@ EmotiBit is a wearable sensor module for capturing high-quality emotional, physi
 
 **Tutorial Coming Soon!**
 
-## HEGduino
```
Comment (retiutut) [inline website/docs/ThirdParty/ThirdPartyLanding.md:17]: 👍

File: website/docs/ThirdParty/WiFiShield/01-Wifi.md [hunk 2]
```diff
@@ -2,9 +2,10 @@
 id: WiFiLanding
 title: OpenBCI WiFi
 ---
-## [BUY IT!](https://shop.openbci.com/collections/frontpage/products/wifi-shield)
 
-WiFi Shields are available for sale [on the OpenBCI store](https://shop.openbci.com/collections/frontpage/products/wifi-shield?variant=44534009550).
+::: warning
```
Comment (retiutut) [inline website/docs/ThirdParty/WiFiShield/01-Wifi.md:6]: Maybe something less harsh than a Warning? Maybe Info?

**Review by retiutut — CHANGES_REQUESTED** [review]
Check that it appears in sidebar. Lower admonition. Maybe add more info on new cEEGRID doc.

**Comment by retiutut** [comment]
From Joe:
cEEGrids are a flexible, re-usable and convenient around-the-ear electrode array that capture EEG and other physiological phenomena. This biosignal acquisition system allows the discrete recording of electrophysiological changes (e.g., EEG, ECG, EMG) from inconspicuous electrodes positioned around the ears.

File: website/docs/ThirdParty/WiFiShield/01-Wifi.md (hunk 2 above)
Comment (jartuso) [reply website/docs/ThirdParty/WiFiShield/01-Wifi.md:6]: yes, please make it any color you think is appropriate

**Review by jartuso — COMMENTED** [review]
changed ::: warning to ::: info

I don't know how to check whether the new page appears in the sidebar

**Review by retiutut — APPROVED** [review]

**Review by retiutut — APPROVED** [review]

**Pull request merged**